autolens/analysis/aggregator/aggregator.py

```diff
@@ -5,49 +5,38 @@
 
 from autofit.database.model.fit import Fit
 from autogalaxy.analysis.aggregator.aggregator import (
-    imaging_via_database_from,
-    interferometer_via_database_from,
+    _imaging_from,
+    _interferometer_from,
 )
 
 from functools import partial
 import numpy as np
 from os import path
 import json
 
-from typing import Optional
+from typing import Optional, List, Generator
 
 
-def tracer_gen_from(aggregator):
+def _tracer_from(fit: Fit, galaxies: List[al.Galaxy]) -> "al.Tracer":

```

Comment:
Quoting a type isn't a subtitute for not importing it. It's really just for when a type is defined further down in the file.

I imagine you got import errors importing autolens as al. This is because doing that requires this line to be executed i.e. it's circular. You can avoid that happening by explicitly importing Tracer from its package.